.env.example

```
@@ -1,2 +1,5 @@
 ## MapTiler API key (create in MapTiler dashboard)
-VITE_MAPTILER_API_KEY="YOUR_MAPTILER_KEY"
+VITE_ENVIRONMENT= local | dev | production
```

dotenv-linter (4.0.0)

[warning] [SpaceCharacter] The line has spaces around equal sign

(SpaceCharacter)